### PR TITLE
[HB-7187] Update UnityAds adapter to perform `setUp` on `IO` instead of `Main`

### PR DESCRIPTION
**Pull request by bwised**
To reduce the potential of ANRs, the adapter `setUp` method has been changed to be perform the work on the `IO` coroutine context.

### CODE REVIEW

**Review by chauduyphanvu — APPROVED** [review]

**Review by bichenwang — APPROVED** [review]

**Comment by bichenwang** [comment]
did we want to fold this into the 4.10.0 adapter?

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 Note the first digit of every adapter version corresponds to the major version of the Chartboost Mediation SDK compatible with that adapter. 
 Adapters are compatible with any Chartboost Mediation SDK version within that major version.
 
+### 4.4.9.2.2
+- Performs initialization on `IO` to help reduce potential ANR issues.
```
Comment (CB-RyanMcCormick) [inline CHANGELOG.md:7]: nit: `Perform` rather than `Performs`

**Review by CB-RyanMcCormick — APPROVED** [review]

**Comment by RayGTile** [comment]
I thought this PR got merged in a long time ago, but apparently it has still been lingering.  I suppose we should make a `4.4.10.0.1` with this then.

**Comment by bwised** [comment]
I thought this PR got merged in a long time ago, but apparently it has still been lingering.  I suppose we should make a `4.4.10.0.1` with this then.

**Comment by CB-RyanMcCormick** [comment]
> I thought this PR got merged in a long time ago, but apparently it has still been lingering. I suppose we should make a `4.4.10.0.1` with this then.

Yup. Want me to do it?

**Comment by bwised** [comment]
I got it.  Already started it. I don't recall where the JIRA ticket is anymore, so I need to make a new one and then get this certified.

File: CHANGELOG.md (hunk 1 above)
Comment (bwised) [reply CHANGELOG.md]: Done.

**Comment by CB-RyanMcCormick** [comment]
Ray, do you want me to merge this and update the adapter pointer in the main repo?

**Pull request merged**